cli/src/print.rs

```diff
@@ -61,6 +61,18 @@ pub(crate) fn print_create_android_success_message() {
     print_footer_message();
 }
 
+pub(crate) fn print_create_web_success_message() {
+    print_green_bold("Template created successfully!".to_string());
+    println!();
+    print_green_bold("Next steps:".to_string());
+    println!();
+    print_green_bold("  You can now use the following command to start web server:".to_string());
+    println!();
+    print_bold(r"    cd web && yarn start".to_string());
+    println!();
+    print_green_bold("This will start a simple web server for your browser.".to_string());
```

Comment:
now we have the page for flutter and react native like
https://github.com/zkmopro/mopro/blob/a95b7f7ee81e8a111b321f4ef763d4d129cede65/cli/src/print.rs#L80

I think we can also maintain this setup page for website

Reply:
Okay, I will add the setup page name as "Web(WASM) Setup" and also add `Prerequisites` for installing `wasm-pack`.